tests/regtest_spec.py

```diff
@@ -1,92 +1,111 @@
 #!/usr/bin/python
 """Print a test spec on stdout.
```

Comment:
Should be matrix_spec.py so that demo.sh run isn't broken ?

For demo.sh to work, we need 3 cases called 'demo-*', run serially.

regtest.sh : Use matrix_spec.py output

demo.sh: Use existing regtest_spec.py